Orange/canvas/application/errorreporting.py

```diff
@@ -187,8 +185,8 @@ def _find_widget_frame(tb):
             widget = frame.tb_frame.f_locals['self'].__class__
             widget_module = '{}:{}'.format(widget.__module__, frame.tb_lineno)
 
-        packages = sorted(["%s==%s" % (i.project_name, i.version)
-                           for i in pip.get_installed_distributions()])
+        packages = ', '.join(sorted(["%s==%s" % (i.project_name, i.version)
```

Comment:
IIUC, sorted can work with a generator so expression ["%s==%s"...] can be passed without brackets.